src/common/dependency-manager/local-manager.ts

```diff
@@ -21,6 +23,92 @@ export default class LocalDependencyManager extends DependencyManager {
     this.config_path = config_path || '';
   }
 
+  protected static loadParameters(dependency_manager: DependencyManager) {
```

Comment:
Any reason this method isn't in the abstract class?

Reply:
The reason was that `LocalServiceNode` isn't accessible to the abstract class. I can move it there, but I didn't think that that really would belong